### DISCUSSION TITLE
Feature Request: support bluetooth connection?

### DISCUSSION THREAD

**Discussion by yhcharles** [debauchee/barrier#1274] [Q&A]
Is it possible to connect two computers via bluetooth connection?

For example, I have two computers connected to same WIFI, but one of them is connected to a VPN network. In this case, barrier won't work. But if barrier can also connect the two computers via bluetooth, it won't be a problem.
This idea seems possible as in a similar software: http://www.acrosscenter.com/

**Comment by eldorel** [comment]
You can setup networking via bluetooth on a lot of devices. 

The bluetooth 'PAN' connection shows up as ethernet devices in the windows network devices. 
Set them with ip addresses from a different subnet to the main nic and configure barrier to use those addresses.

**Comment by p12tic** [comment]
It seems that the last comment has answered why this behavior is seen and that it's not a bug. Therefore I'm closing this issue. If I misunderstood, please reopen.

**Comment by ncake** [comment]
To anyone looking this up: I got a Bluetooth PAN connection to work, and it really really sucked. The bandwidth is too small for this app to work smoothly, even with encryption turned off.

**Comment by Vaisakhkm2625** [comment]
can barrier act as a bluetooth mouse or keyboard?
